themes/bootstrap5/scss/components/search.scss

```diff
@@ -119,12 +119,77 @@ input[type="search"]::-webkit-search-decoration {
   }
 }
 
-.searchHomeContent { @include clearfix(); }
+.searchHomeContent {
+  @include clearfix();
+}
+
+.home-facet {
+  @media (min-width: 768px) {
+    flex: 0 1 100%;
+  }
+}
+
+.search-home-facets,
+.home-facet-container {
+  @media (min-width: 768px) {
+    display: flex;
+  }
+}
+
+.home-facet-container {
+  padding-right: $home-facet-container-right-padding; // set left padding here to define distance between containers for visual division
+
+  @media (min-width: 768px) {
+    height: 100%; // required to make visual division line go full height, when 'column-fill: balance;' is set on '.two-columns'
+  }
+}
+
 .home-facet-list {
-  padding-left: 0;
-  padding-right: 1rem;
-  list-style: none;
+  list-style: $home-facet-list-list-style; // defines the list style (default = 'none')
+  overflow: $home-facet-list-column-overflow; // Defines what's to be done if there are more entries than fit the columns
+  padding-left: $home-facet-list-left-padding; // overwrites default left-padding for <ul>s
+  padding-right: $home-facet-list-right-padding;
+
+  @media (min-width: 768px) {
+    border-right: $home-facet-list-border-right-style; // adds visual division between containers for devices >=768px (containers are stacked on smaller devices)
```

Comment:
Would [`column-rule`](https://css-tricks.com/almanac/properties/c/column-rule/) be a better fit for this? Would prevent the need for min-width and last-of-type (below). [96% browser availability](https://caniuse.com/mdn-css_properties_column-rule).